sourcemash/api/categories.py

```diff
@@ -13,18 +22,46 @@
 }
 
 category_status_fields = {
-  'item_count': fields.Integer,
-  'unread_count': fields.Integer
+  'unread': isUnread
 }
 category_status_fields = dict(category_fields, **category_status_fields)
 
+category_status_count_fields = {
+  'item_count': fields.Integer,
```

Comment:
We do it for feeds so is it weird not to do for categories? That being said, I’m not too attached and wouldn’t have any problem cutting it.